Assets/Scripts/PlayerMovementPlayback.cs

```diff
@@ -37,7 +37,7 @@ void FixedUpdate()
             if(line != null)
             {
                 string[] data = line.Split(' ');
-                if (!data[0].Equals("EndTaskCompleted") && data.Length > 1) //If this line has coordinates (not the end task tag)
+                if (!data[0].Equals("EndTaskCompleted") && data.Length > 2) //If this line has coordinates (not the end task tag)
```

Comment:
more thorough explanation of what that number means is needed I think so we all understand